src/zzz_od/operation/choose_predefined_team.py

```
@@ -1,124 +1,584 @@
+                if (
+                    abs(mr.center.x - center.x)
+                    <= ChoosePredefinedTeam.SELECT_BUTTON_FEEDBACK_HALF_WIDTH
+                    and abs(mr.center.y - center.y)
+                    <= ChoosePredefinedTeam.SELECT_BUTTON_FEEDBACK_HALF_HEIGHT
+                ):
+                    return True
+        return False
+
+    @staticmethod
+    def _is_selected_text(normalized_text: str) -> bool:
+        """统一已选按钮的 OCR 结果，仅在目标卡片的按钮区域内使用。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)